transmartApp/grails-app/conf/logback.groovy

```diff
@@ -50,34 +53,43 @@ if (productionMode && logDirectory) {
     root(WARN, ['transmart'])
 }
 
+
 /**
  * Configuration for writing audit metrics.
  * This needs to be placed in the out-of-tree Config.groovy, as the log4j config there will override this.
- * (and don't forget to 'import org.apache.log4j.DailyRollingFileAppender',
- * 'import org.transmart.server.logging.ChildProcessAppender' and 'import org.transmart.server.logging.JsonLayout'.)
+ * See https://logback.qos.ch/manual/appenders.html for details on configuration
  */
 /*
+appender('fileAuditLogger', RollingFileAppender) {
```

Comment:
Why is this stuff in the transmartApp logback config, can that file not be removed completely?